repos/aspnet-razor.proj

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
+  <PropertyGroup>
+    <RazorSdkProjectDir>$(ProjectDirectory)src/Microsoft.NET.Sdk.Razor/</RazorSdkProjectDir>
+    <RazorSdkProjectFile>$(RazorSdkProjectDir)Microsoft.NET.Sdk.Razor.csproj</RazorSdkProjectFile>
+
+    <BuildNumber>30932</BuildNumber>
+
+    <BuildCommandArgs>pack</BuildCommandArgs>
+    <BuildCommandArgs>$(BuildCommandArgs) $(RazorSdkProjectFile)</BuildCommandArgs>
+    <BuildCommandArgs>$(BuildCommandArgs) /p:Configuration=$(Configuration)</BuildCommandArgs>
+    <!-- A nonfinal build adds "-t000" to the package version. (Prerelease builds are "final".) -->
+    <BuildCommandArgs>$(BuildCommandArgs) /p:IsFinalBuild=true</BuildCommandArgs>
```

Comment:
Should this be equivalent to one of our `DropSuffix` or `Stabilized` properties?

Reply:
My intent behind the comment is to explain why it shouldn't be. 😄 (Specifically, `UseStableVersions` in dependencies.props.) I'll make this clearer.

Reply:
Wait a sec... there's an issue for 2.2 where I ended up building `2.2.0-preview3-final` even though ProdCon built `2.2.0-preview3-35497`. I talked with @natemcmaster about it, who said `dotnet pack /p:BuildNumber=xyz /p:IsFinalBuild=true` (like it does now) should include the build number, but my 2.2 build just got to the point where I can see this still produces `2.2.0-preview3-final`.

Looking closer at version.props, it looks like `IsFinalBuild` should actually be `false`. Then, `PackageVersion` won't be set, and the SDK is free to combine `VersionPrefix` and `VersionSuffix`.

Reply:
For 2.2 we stopped doing the `IsFinalBuild=true` step for previews, as you'll see on nuget.org: https://www.nuget.org/packages/Microsoft.NET.Sdk.Razor/2.2.0-preview3-35497.

But this PR is targeting 2.1, right? So, `IsFinalBuild=true` should be set for an RTM build of 2.1.x